### ISSUE TITLE
About reproduction of experimental results

### ISSUE THREAD

**Issue by adam99goat** [ShuweiShao/AF-SfMLearner#20]
Strongly grateful for your excellent work and sorry to bother you for some puzzle. I tried to retrain the network to acquire the same results of SCARED as yours but failed. The validation of end-to-end and stage-wise is shown below.
![end](https://user-images.githubusercontent.com/49334034/226329322-9d54b5db-8390-411d-8394-0f853d461a7e.png)
![stage](https://user-images.githubusercontent.com/49334034/226329335-80ff70ac-3e4f-45ae-9532-37d6405d7871.png)
Though the result is worse than what are reported in your paper, the experimental result of Monodepth2 by myself is strangely better. The abs_rel value in your paper is 0.071. 
![image](https://user-images.githubusercontent.com/49334034/226331155-c04eb366-4330-4ec8-a146-df903ef7108b.png)
The data preparation is based on OpenCV which leavage the same video encoder and decoder as ffmpeg. So I can't figure out why but only ask you for favour. Look forward to your reply and thank you again!

**Comment by adam99goat** [comment]
Excited and thankful for your reply. My pytorch version is 1.8.0. And I would like to know your version if convenient. I will try your valuable suggestion and retrain the net in stage-2 with help of provided weights of OFNet. Thank you a lot!

**Comment by ShuweiShao** [comment]
My PyTorch is 1.2.0 and torchvision is 0.4.0.

**Comment by adam99goat** [comment]
> My PyTorch is 1.2.0 and torchvision is 0.4.0.

What a pity I think there still exist some problems. Maybe the data preparation should be based on ffmpeg. So could you share related code or command of pre-processing if convenient? Thanks again!

**Comment by ShuweiShao** [comment]
Hi，I can't find the preprocessing code now since a long time has passed,  maybe you can search for the usage of ffmpeg.

**Comment by adam99goat** [comment]
I feel so sorry for my late reply. Actually when I followed the 2-stages custom training with the given weight of OFNet, the result is shown below.
![image](https://user-images.githubusercontent.com/49334034/230763245-7f819430-ecf7-49bc-abc2-6e59fd088c00.png)
Also PyTorch 1.2.0 and torchvision 0.4.0.

**Comment by cccrystalyang** [comment]
Hi,  I also encountered the same problem. I have preprocessed the dataset using the author's update method, and the code 'align_corner' are also set to True. The same results as mentioned in the paper can be obtained using the weights provided by the author, but the results of retraining DepthNet with the given OFNet weights also remain around 0.061. May I ask what optimizations and adjustments have you made?
